scripts/plotting/global_latlon_vect_map.py

```diff
@@ -412,10 +417,12 @@ def global_latlon_vect_map(adfobj):
                                 # I'll just call this one "LatLon_Mean"  ... would this work as a pattern [operation]_[AxesDescription] ?
                                 plot_name = plot_loc / f"{var_name}_{lv}hpa_{s}_LatLon_Vector_Mean.{plot_type}"
 
-                                #Remove old plot, if it already exists:
-                                if plot_name.is_file():
+
+                                # Check redo_plot. If set to True: remove old plot, if it already exists:
+                                if (not redo_plot) and plot_name.is_file():
+                                    continue
+                                elif (redo_plot) and plot_name.is_file():
                                     plot_name.unlink()
-                                #End if
```

Comment:
In general I wouldn't remove these `#End if` comments, as they can help future users better understand where the if-statement has been closed.